app/daemon_portforward.go

```diff
@@ -0,0 +1,227 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package app
+
+import (
+	"context"
+	"io"
+	"net"
+	"strconv"
+
+	"github.com/mendersoftware/go-lib-micro/ws"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/mendersoftware/mender-connect/connectionmanager"
+)
+
+const (
+	ProtoTypePortForward       = 3
```

Comment:
Maybe these top-level aliases should be grouped in a master file so we don't risk picking the same number? Git-merging could easily mess this up behind our backs.

Reply:
This was temporary, mainly because go-lib-micro wasn't ready.
They moved there, now. :)